cmd/deploy/remote.go

```diff
@@ -116,6 +124,12 @@ func (rd *remoteDeployer) Deploy(ctx context.Context, deployOptions *Options) er
 		return fmt.Errorf("failed to create ignore rules for remote deploy: %w", err)
 	}
 
+	var ctxPath string
+	if deployOptions.Manifest.Deploy != nil {
+		manifestPathDir := filepath.Dir(deployOptions.ManifestPathFlag)
+		ctxPath = path.Clean(path.Join(cwd, manifestPathDir, deployOptions.Manifest.Deploy.Context))
```

Comment:
If I'm not wrong, `cwd` here should already be pointing to the dir where the manifest is located, right @jLopezbarb ?

Reply:
Why? cwd it's retrieved using the function `GetOriginalCWD` which returns the cwd that the user has used

Reply:
> Using the vanilla cwd fixes the issue given that at some point I assume we are changing the workdir to where the manifest is located.

Yes, deploy, destroy and tests commands call to this function `UpdateCWDtoManifestPath` which updated the CWD to the folder where the Okteto Manifest is located